transmogrifier/helpers.py

```diff
@@ -69,7 +68,7 @@ def parse_date_from_string(
     """
     for date_format in DATE_FORMATS:
         try:
-            return datetime.strptime(date_string, date_format)
+            return datetime.strptime(date_string, date_format).astimezone(UTC)
```

Comment:
I don't think we need to worry about the repercussions of this, but noting this could affect the timestamps derived.  If we had logic based on these timestamps, or if they were financial/audit/etc. information I think it'd be a bigger deal.  But given this is just parsing date strings from metadata, a difference of a couple of hours from previous runs seems unimportant.

Reply:
Good point and agree it's probably not as big of a deal here as opposed to `alma-creditcardslips` or `alma-sapinvoices`